interview-questions.md

```diff
@@ -5,57 +5,73 @@ Answer the following questions. First, without external resources. Challenge you
 1. Why would you use the method super()?
 
   Your answer:
+  the super() method allows us to more quickly create class extensions without repeatedly entering constructors that would be used throughout all of them.
 
   Researched answer:
-
+  you would use the super method to access and call on functions from an object's parent.
 
 
 2. What is a virtual DOM?
 
   Your answer:
+  A virtual DOM (Document Object Model) off the top of my head, I think it's the model in which browsers see our data.
 
   Researched answer:
-
+  In React every DOM object has a corresponding "virtual DOM" that has the same properties but does not have the power to directly change what's on the screen. 
 
 
 3. When creating a basic (stateless) class component in React, what are the necessary elements needed to display "Hello World" in the browser?
 
   Your answer:
+  Not sure if I'm interpreting the question properly but I think we need everything we want to return in the component in one div, and we also need the usual class elements like constructors and methods.
```

Comment:
You've pretty much got the right idea. So generally we'd say something like "I'd need to import react and component, then i'd need to define a class, then i'd need a return and a render, an opening and closing div then i'd create a text tag inside the div. Finally i'd export the component." 

How you answered is fine, just wanted to give you something to think about.